packages/app/src/cli/models/extensions/functions.ts

```diff
@@ -44,6 +44,7 @@ export class FunctionInstance<TConfiguration extends FunctionConfigType = Functi
   entrySourceFilePath?: string
   configuration: TConfiguration
   configurationPath: string
+  _usingExtensionsFramework: boolean
```

Comment:
Why the underscore? 🤔 Is this some js/ts idiom I'm missing?

Reply:
TIL!

Reply:
You can use `private usingExtensionsFramework: boolean`
It's easier to understand :)